packages/dialect-react-ui/entities/wrappers/ConnectionWrapper.tsx

```diff
@@ -29,8 +35,8 @@ export default function ConnectionWrapper({
   } = useDialectConnectionInfo();
 
   // FIXME: trigger some fetch from some of backends to get connection state
-  const { errorFetchingThreads } = useThreads({
-    refreshInterval: 5000,
+  useThreads({
```

Comment:
Did you test it, does it produce error screen in case of network error?

Reply:
No, I haven't explicitly, but I've only removed the unused variable. If it's necessary to make a request, this would do one, wouldn't it?